packages/contracts-bedrock/contracts/dispute/DisputeGameFactory.sol

```diff
@@ -164,4 +184,20 @@ contract DisputeGameFactory is OwnableUpgradeable, IDisputeGameFactory, IVersion
         gameImpls[gameType] = impl;
         emit ImplementationSet(address(impl), gameType);
     }
+
+    /**
+     * @dev Packs an address and a uint256 into a single bytes32 slot.
+     */
+    function _packSlot(address _addr, uint256 _num) internal pure returns (bytes32 slot_) {
+        slot_ = bytes32(_num << 160) | bytes32(uint256(uint160(address(_addr))));
+    }
```

Comment:
Can we fuzz test this?

Reply:
+1. 
You could basically copy, paste and lightly modify the [Aliasing invariant tests](https://github.com/ethereum-optimism/optimism/blob/7015d66c9f5302a92b0ead60dc41fd05d350ade6/packages/contracts-bedrock/contracts/test/invariants/AddressAliasHelper.t.sol#L1).